Source/CSBackend/Rendering/OpenGL/Base/RenderCommandProcessor.cpp

```diff
@@ -365,10 +365,7 @@ namespace CSBackend
             glDepthMask(GL_TRUE);
             
             glClearColor(renderCommand->GetClearColour().r, renderCommand->GetClearColour().g, renderCommand->GetClearColour().b, renderCommand->GetClearColour().a);
-            glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
-            
-            glBlendEquation(GL_FUNC_ADD);
```

Comment:
GLBlendEquation has been moved into material. It was one of the few things we didn't expose so figured I'd move it at the same time. glClear should still be there as we have to clear the stencil bit too. Will look into where that went